public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,93 @@
+# Setup Google Drive remote
+
+To setup Google Drive remote you should
+[create and configure Google Project](#create-and-configure-google-project) if
+you don't have one yet.
+
+## Create and Configure Google Project
+
+Since Google Drive has tight API usage quotas, one time creation and
+configuration of your own `Google Project` is required. Required `client id` and
+`client secret` can be generated only once and shared with other DVC users in
+your organization.
```

Comment:
`client id` and `client secret`, I guess, can be used as common notion.
`client_id` and `client_secret` are used as placeholders for exact values only withing cli blocks.

Reply:
> `` Since Google Drive has tight API usage quotas, one-time creation and configuration of a _Google Project_ is required. The required parameters, `client id` and `client secret`, can be generated only once. Please share them with other DVC users in your team to grant them access to this storage. ``

@jorgeorpinel thanks!
Since client id and client secret only identifies what Google Project uses Google's API, I've updated your text as following:

``Since Google Drive has tight API usage quotas, one-time creation and configuration of a _Google Project_ is required. The mandatory credentials, `client id` and `client secret`, can be generated only once. Please share them with other DVC users in your team to grant them access to Google's API.``

Reply:
OK I just wouldn't use backquotes for common notation i.e. just say "client ID". But probably best to always use `client_id` to avoid confusion. Anyway, I'm going to get your branch and make some changes myself. Will let you know for a final review (: